fineract-client/build.gradle

```diff
@@ -72,26 +72,23 @@ task buildTypescriptAngularSdk(type: org.openapitools.generator.gradle.plugin.ta
     skipValidateSpec = true
     inputSpec = "file:///$swaggerFile"
     outputDir = "$buildDir/generated/typescript".toString()
-    apiPackage = 'org.apache.fineract.client.services'
-    invokerPackage = 'org.apache.fineract.client'
-    modelPackage = 'org.apache.fineract.client.models'
+    apiPackage = 'apache-fineract-client/services'
+    invokerPackage = 'apache-fineract-client/invoker'
+    modelPackage = 'apache-fineract-client/models'
     configOptions = [
-        apiModulePrefix: 'fineract',
-        configurationPrefix: 'fineract',
+        apiModulePrefix: 'apacheFineractClient',
+        configurationPrefix: 'apacheFineractClient',
         ngVersion: '12.0.0',
-        npmName: 'apache-fineract-client'
+        npmName: '@apache/fineract-client',
+        npmRepository: "${npmRepository}"
     ]
     dependsOn = [
         ':fineract-provider:resolve'
     ]
 }
 
-licenseFormatMain.dependsOn buildJavaSdk
```

Comment:
Wait a sec. Are you sure this is gonna be replaced by the change below?

When I was fixing the incremental build for Gradle, this combination worker flawlessly. Have you tested whether it's broken?